### DISCUSSION TITLE
from_statoil error

### DISCUSSION THREAD

**Discussion by minzhangut** [PMEAL/OpenPNM#2383] [Q&A]
When I use openpnm 2.8.2, I got the error of "AttributeError: module 'openpnm.io' has no attribute 'from_statoil'". Then I tried the openpnm dev version (2.8.2.dev1116), I got the error of "ValueError: Length mismatch: Expected axis has 7 elements, new values have 4 elements". I think it might because from_statoil only supports the old version of maximal ball approach outputs?

**Comment by jgostick** [comment]
The statoil for sure only supports the old format.  I am not sure, but I think there is some interoperability between the statoil and avizo, which uses the pergeos format...you could try that.  Otherwise, I won't be able to solve this for quite a while..  Writting an IO class takes ages and I'm already maxed out on coding tasks.

**Reply by minzhangut** [reply]
I think the new file formats are consistent with the old ones, and I just modified the new output files by mistake.

**Reply by jgostick** [reply]
Does that mean your problem is solved?

**Reply by minzhangut** [reply]
Yes I confirmed.
